README.md

```diff
@@ -50,15 +50,25 @@ To clone the repository, complete the following steps:
 
 ## Get started
 
-For an overview of the user interface and features, refer to
-the [Google Home Sample App for Matter Guide](https://developers.home.google.com/samples/matter-app).
-
-To review code samples and start building, refer to
-the [Build an Android App for Matter](https://developers.home.google.com/codelabs/matter-sample-app)
-Codelab.
-
-To make sure that your device has the latest Matter support, review the
-[Verify Matter Modules & Services](https://developers.home.google.com/matter/verify-services) guide.
+*   For an overview of the user interface and features, refer to
+    the [Google Home Sample App for Matter Guide](https://developers.home.google.com/samples/matter-app).
+*   To review code samples and start building, refer to
+    the [Build an Android App for Matter](https://developers.home.google.com/codelabs/matter-sample-app)
+    Codelab.
+*   To make sure that your device has the latest Matter support, review the
+    [Verify Matter Modules & Services](https://developers.home.google.com/matter/verify-services)
+    guide.
+*   [Create a Matter integration](https://developers.home.google.com/matter/integration/create) and
```

Comment:
[Create a Matter integration] in the Home Developer Console and build...

Reply:
Thanks - pushed an update; LMKWYT.